<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Dynamic Systems Biology Group</title>
  <meta name="description" content="Dynamic Systems Biology Group">
  <meta name="author" content="Marcelo S. Reis">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="icon" type="image/svg" href="images/king-of-hearts.svg">

</head>


<body>


  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="twelve columns" style="margin-top: 2%">
        <h4>Dynamic Systems Biology Group</h4>
    </div>
  </div>


<div class="navBar" id="mainNavBar">
  <a href="index.html">About</a>
  <a href="members.html">Members</a>
  <a href="publications.html">Publications</a>
  <a href="projects-and-software.html">Projects and Software</a>
  <a href="teaching.html">Teaching</a>
  <a href="contact.html">Contact</a>
  <a href="javascript:void(0);" class="icon" onClick="openDrawerMenu()">&#9776;</a>
  <!--&#9776; is the code for the 3 line menu button-->
</div>

<div class="container">
  <div class="row" style="margin-top: 2%">
  <h5> Welcome! </h5>
  
  The Dynamic Systems Biology Group, led by <a href="members.html#msreis">Marcelo S. Reis</a> and hosted at the Cell Cycle Laboratory, <a href="http://butantan.gov.br/instituto-butantan/about-us" target="_blank">Butantan Institute</a>, is a team of scientists whose main goal the development of cutting-edge Bayesian Model Selection and Machine Learning techniques and their application into the modeling of biological systems, with emphasis to their underlying dynamic mechanisms. Currently, we are focused on three research lines: the modeling of cell signaling pathways in the context of cancer biology; the deciphering of the DNA replication dynamics in trypanosomatids; the inference of phyloproteomic cladograms (trees) from mass spectrometry (MS)-based proteomics assays.
  
  <br/><br/><br/>
  
  <div class="container">
    <div class="row">
      <h5>Modeling of cell signaling pathways in the context of cancer biology</h5>
    </div>
     <div class="row">
    Cancer cells proliferate relentlessly, partly due to deregulation of several cell signaling pathways. Once there are technical limitations in wet lab screening procedures, dynamic modeling of signaling pathways is an important tool to predict molecular and ultimately cell phenotype response following cellular stimuli or chemical species perturbations (e.g., protein inhibition or activation). Therefore, our group investigates novel methods to model, using systems of ordinary differential equations (ODEs), highly-predictive cell signaling pathways, focusing on both mitogenic and stress-mitigation pathways, which are relevant for cancer cell phenotype. One of our current projects in this research line is "<a href="projects-and-software.html#Goldilocks">Finding the "Goldilocks zones" of cell signaling pathways in cancer therapy</a>", which is carried out in collaboration with Prof. Hugo Armelin (Coordinator of the FAPESP RIDC CeTICS center), Dr. Juliane Liepe (Group Leader at Max Planck Institute for Biophysical Chemistry, Germany), Prof. Junior Barrera (Director of Institute of Mathematics and Statistics, University of São Paulo) and Prof. Ronaldo Hashimoto (Associate Professor at Institute of Mathematics and Statistics, University of São Paulo).
    </div>
    <div class="row">
        <br/>
        <img src="images/Reis_2017_Fig6.jpeg" class="bg" alt="Figure 6 of Reis, Noël et al. (2017)">
    </div>
    <div class="row">
        <figure>
          <img src="images/Reis_2017_Fig7.png" class="bg" alt="Figure 7 of Reis, Noël et al. (2017)"><br/>
          <figcaption class="caption"><i>The ERK kinase mitogenic signaling pathway in mouse Y1 cancer cells. A dynamic model that includes a negative feedback from ERK to Raf kinase (top) is able to explain the ERK kinetics in FGF2-stimulated Y1 cells (bottom). Results extracted from Figures 6 and 7 of <a href="publications.html#Reis:2017b">Reis, Noël et al. (2017).</a></i></figcaption>
        </figure>
    </div>
  </div>

    <hr><br/>
  
  <div class="container">
    <div class="row">
      <h5>Deciphering of the DNA replication dynamics in trypanosomatids</h5>
    </div>
    <div class="row">
    Among the Eukaryota domain, the protozoan are comprised of single-celled organisms that might be either free-living or parasitic. The trypanosomatids fit in the latter category, with many clinically relevant species, such as <em>Trypanosoma cruzi</em> (pathogen that causes the Chagas disease) and <em>T. brucei</em> (responsible for the sleeping sickness). Once there is no known cure for the diseases caused by those parasites, the study of their biology is a subject of immediate biomedical interest. In this regard, once those parasites alternate between replicative and non-replicative forms, the study of their DNA replication dynamics is a relevant topic. However, those organisms have a peculiar genomic organization, with their genes arranged into polycistronic clusters and the presence of constitutive transcription along the whole cell cycle. Those genomic properties yield a DNA replication dynamics in those biological systems that is not directly comparable to higher eukaryotes (e.g., mammalian cells). Therefore, in this research line, we propose the designing, data-based calibration and simulation of stochastic dynamic models of the DNA replication dynamics of trypanosomatids. We are particularly interested in the impact of the conflicts between DNA replication and transcription machineries on the DNA replication programming of those parasites. Presently we have in this research line the project "<a href="projects-and-software.html#replicationModels">DNA replication programming in trypanosomatids</a>", which is carried out in collaboration with Dr. Maria Carolina Elias (Director of Cell Cycle Laboratory, Butantan Institute). 
    </div>
    <div class="row">
      <br/>
      <figure>
        <img src="images/daSilva_2019_Fig3A.jpeg" class="bg" alt="Figure 3A of da Silva et al. (2019)"><br/>
        <figcaption class="caption"><i>An example with two replication origins (theta 1 and theta 2) of how our dynamic model simulates the replication-transcription conflicts in <em>T. brucei</em>. When we simulate replication and transcription at the same time, if there is a head-to-head collision between replication and transcription machineries (replisome and RNA polymerase, respectively), then the replication fork collapses, leading to an unbinding of its respective replisome and the involved RNAP. Results extracted from Figure 3A of <a href="publications.html#daSilva:2019">da Silva et al. (2019).</a></i></figcaption>
      </figure>
    </div>
  </div>

    <hr><br/>
  
  <div class="container">
    <div class="row">
        <h5>Inference of phyloproteomic trees from MS-based proteomics assays</h5>
    </div>
    <div class="row">
    Snakebite envenomation is a life-threatening accident that consists in the injection into a victim of an certain amount of venom through the bite of a snake. It is considered by the World Health Organization as <a href="https://www.who.int/snakebites/disease/en/" target="_blank">a neglected tropical disease since June 2017</a>. Since snake venoms are complex mixtures, being composed mostly of proteins and peptides, the investigation of snake venoms through proteomic profiling comparisons among species are relevant for the development of more efficient anti-venom treatments (e.g., by selecting molecular targets that are shared by a number of venoms). Moreover, the proteomic profiling comparison through phyloproteomic cladograms (tree) is also very important in the context of evolutionary studies, since they might help us to understand the interplay between venom composition, phylogeny and environment. However, to yield trustworthy phyloproteomic trees is a challenging task due to limitations of mass spectrometry (MS)-based proteomics, the analytical strategy that is used for snake venom proteomic profiling. One of those limitations is the sequence database-based peptide and protein identification, a step that is limited by the completeness of the used database. Therefore, in this research line we aim to develop methods to circumvent MS-based limitations to infer reliable phyloproteomic trees; to this end, we are dealing directly with MS raw data, skipping the database identification step altogether, and developing multi-resolution-based approaches to generate occurrence matrices for a Bayesian inference of phyloproteomic trees. The current project in this research line is "<a href="projects-and-software.html#phyloproteomics">Phyloproteomic analysis of snake venom proteomes yielded by mass spectrometry-based assays</a>", in collaboration with Dr. Solange Serrano (Principal Investigator at Applied Toxinology Laboratory, Butantan Institute).
    </div>
    <div class="row">
        <br/>
        <figure>
          <img src="images/gmaciel_term_paper.jpeg" class="bg" alt="Gustavo Maciel's term paper (2019)">
          <figcaption class="caption"><i>Fluxogram of the methodology under development. Snake venoms from different species are analyzed in a mass spectrometer, yielding a raw file, one per venom. Each yielded file can be used to produce an ion map; those maps can be aligned among venoms. The aligned maps then can be used to define a partition scheme that should be applied into each ion map; the chosen partition defines the resolution that will be used at each region of the ion map for the generation of the NEXUS files. Using these latter files, the phyloproteomic tree can be estimated through the Bayesian inference procedure implemented in the MrBayes software. Finally, the mtDNA-based phylogenetic tree also generated using MrBayes can be compared with the phyloproteomic one using statistical tests like CADM. Figure extracted from Gustavo Maciel's term poster (2019).</i></figcaption>
        </figure>  
    </div>
  </div>

  <br/><hr><br/>

If you became interested in one of those research lines, please check it out at the <a href="projects-and-software.html">Projects and Software</a> page to know about the projects that are being carried out by our group, as well as the in-house software that we develop to tackle them. Feel free to <a href="contact.html">contact us</a> for any question or inquiry about opportunities at our group.

 </div>
</div>

<br/><br/>

<footer>
  <div class="container">
        <i><a href="index.html">Dynamic Systems Biology Group</a>, <a href="http://butantan.gov.br/instituto-butantan/about-us" target="_blank">Butantan Institute</a><br/>
        Phone: +55-11-2627-3755, Fax: +55-11-2627-9731<br/>
        marcelo dot reis at butantan dot gov dot br</i>
  </div>
</footer>

<script>
  function openDrawerMenu(){
    var x = document.getElementById("mainNavBar");
    if (x.className === "navBar"){
      x.className += " responsive";
    } else {
      x.className = "navBar";
    }
  }
</script>


<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->


</body>

</html>


