<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Dynamic Systems Biology Group</title>
  <meta name="description" content="Dynamic Systems Biology Group">
  <meta name="author" content="Marcelo S. Reis">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="icon" type="image/svg" href="images/king-of-hearts.svg">

</head>


<body>


  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="twelve columns" style="margin-top: 2%">
        <h4>Dynamic Systems Biology Group</h4>
    </div>
  </div>


<div class="navBar" id="mainNavBar">
  <a href="index.html">About</a>
  <a href="members.html">Members</a>
  <a href="publications.html">Publications</a>
  <a href="projects-and-software.html">Projects and Software</a>
  <a href="contact.html">Contact</a>
  <a href="javascript:void(0);" class="icon" onClick="openDrawerMenu()">&#9776;</a>
  <!--&#9776; is the code for the 3 line menu button-->
</div>


  <div class="container">
    <div class="twelve columns" style="margin-top: 2%">
        <h5>Ongoing projects</h5>
 
 In this page we list the main research projects that are currently in execution by our group. If you are interested in working on one of them, either as a student or a collaborator, do not hesitate <a href="contact.html">to contact us!</a>

 <br/><br/><br/>
 
 </div>    
    
    
<div class="container">
  <div class="row">
    <div class="twelve columns" id="modelSelection">
    <h6><strong>Development of ABC-based approaches to tackle the lack of isolation problem in signaling pathway</strong></h6>
    Dynamic models based on ordinary differential equations (ODEs) are useful tools for investigation of cell signaling pathways. Such models allow us to make predictions about the pathway behavior under different stimuli. ODE-based models are often calibrated using experimental data obtained by probing one or more species at several time points after a given cell stimulation. To this end, Bayesian approaches such as the approximate Bayesian computation (ABC) are relevant tools, since they allow the assessment of the uncertainty in the estimation and also are a natural way to carry out model selection. In general, those models are estimated assuming that they are relatively isolated systems with known inputs; however, in many practical situations, we do not have any guarantee that that assumption holds. Therefore, in this project, we propose to tackle the lack of isolation problem in signaling pathway modeling through the development of a new ABC-based approach, which would take into account that model inputs also must be inferred. To this end, we also intend to use the reduction of the model selection problem to the feature selection problem that was recently developed by our group. Finally, we plan to test the new method on artificial instances and also on experimental measurements that are yielded in cancer cells studies at our lab. We expect that the proposed method will become a relevant tool in the studying of cell signaling pathways in many biological contexts, specially in cancer biology studies.
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#rnogueira">Ronaldo Nogueira</a> - PhD Student <br/><br/>
    <a href="members.html#gestrela">Gustavo Estrela</a> - MSc Student <br/><br/>
    <a href="https://www.mpibpc.mpg.de/liepe" target="_blank">Juliane Liepe, PhD</a> - Associate Investigator<br/>
    <em>(Group Leader at Quantitative and Systems Biology, Max Planck Institute for Biophysical Chemistry, Germany)</em><br/><br/>
    <a href="http://lattes.cnpq.br/8341719454584768" target="_blank">Prof. Hugo A. Armelin</a> - Associate Investigator<br/>
    <em>(<a href="http://www.cetics.com.br/en/" target="_blank">CeTICS Center</a> Coordinator, Butantan Institute)</em><br/><br/>

    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    This project is currently supported by a PhD scholarship from CAPES. Initial parts of this work were supported by <a href="https://bv.fapesp.br/en/bolsas/175684/identification-of-cell-signaling-pathways-based-on-biochemical-reaction-kinetics-repositories/" target="_blank">the MSc scholarship # 2017/20575-9 from FAPESP</a>, and also by <a href="https://bv.fapesp.br/en/bolsas/189321/an-approximate-bayesian-computation-based-approach-to-tackle-the-lack-of-isolation-problem-in-signal/" target="_blank">the BPE grant # 2019/20025-4 from FAPESP</a>; activities of this latter grant were carried out by the Coordinator at Dr. Liepe's group in Göttingen, Germany.
    </div>
  </div>
</div>

<br/><hr>

<div class="container" id="replicationModels">
  <div class="row">
    <div class="twelve columns">
    <h6><strong>DNA replication programming in trypanosomatids</strong></h6>
    Trypanosomatids are protozoa that bear a peculiar genomic topology, with genes organized into polycistrons and constitutive transcription along the whole cell cycle. Given those properties, one hypothesis is that conflicts between DNA replication and transcription machineries contribute for the firing of replication origins along the cell cycle S phase in those organisms. In fact, based on a DNA replication model proposed for human cells, we developed a DNA replication dynamic model for <em>Trypanosoma brucei</em> that was calibrated with published MFA-seq data and used it to predict that increasing constitutive transcription levels indeed incur in an increase of fired origins; however, such increase has little impact on the overall S phase duration (<a href="publications.html#daSilva:2019">da Silva et al., 2019</a>). Those predictions were validated experimentally in alpha-amanitin-based transcription inhibition assays (<a href="publications.html#daSilva:2019">da Silva et al., 2019</a>). Nevertheless, the aforementioned hypothesis was not explored in other trypanosomatids, especially in <em>T. cruzi</em>. Recently, Dr. Maria Carolina Elias' group concluded the DNA origins mapping for <em>T. cruzi</em> through MFA-seq assays. Those assays unveiled that several origins are located in coding regions of the dispersed gene family 1 (DFG-1) genes, a gene family relevant for the parasite life cycle. Once DFG-1 genes have high genetic variability, one possibility is that conflicts between DNA replication and transcription machineries are responsible for the verified genetic variability. If this latter assumption is correct, one corollary is that the origin firing distribution in <em>T. cruzi</em> is conditioned by the parasite's own genomic organization. Therefore, through a combination of dry and wet lab assays, we seek in this project to answer the following questions: 
    
    <br/><br/>
    
    <ul> 
      <li> <em> Do conflicts between DNA replication and transcription machineries induce increased origin firing in T. cruzi?</em> </li>
      <li> <em> Is the origin firing distribution in <em>T. cruzi</em> conditioned by the genomic organization?</em></li>
    </ul>

    Apart from the resolution of the aforementioned questions, we aim with this project to improve computational dynamic models of the DNA replication programming. We also expect theoretical advances (e.g., closed solutions of stochastic processes that describe DNA replication dynamics in trypanosomatids), which would be derived through analyses of computational assays.
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#vhariki">Victor Hariki</a> - Undergrad Student <br/><br/>
    <a href="members.html#bscholl">Bruno Scholl</a> - Undergrad Student <br/><br/>
    <a href="members.html#gcayres">Gustavo Cayres</a> - Undergrad Student (Former Member)<br/><br/>
    <a href="http://lattes.cnpq.br/4253838966347741" target="_blank">Maria Carolina Elias, PhD</a> - Associate Investigator<br/>
    <em>(Principal Investigator at Cell Cycle Laboratory, Butantan Institute)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4775859232176039" target="_blank">Marcelo Santos</a> - Postdoc fellow under Dr. Elias supervision (Former Member)<br/><br/>
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    Initial stages of this project were supported by <a href="https://bv.fapesp.br/en/bolsas/168770/design-and-simulation-of-computational-models-of-the-dna-replication-dynamics-in-kinetoplastids/" target="_blank">the Undergrad Research scholarship # 2016/17775-3 from FAPESP</a>.
    </div>
  </div>
</div>

<br/><hr>


<div class="container" id="phyloproteomics">
  <div class="row">
    <div class="twelve columns">
    <h6><b>Phyloproteomic analysis of snake venom proteomes yielded by mass spectrometry-based assays</b></h6>

    Snake venoms are complex mixtures composed mostly of proteins and peptides. One analytical strategy that is often used to study them is the mass-spectrometry (MS)-based proteomics. Recent analysis using proteomic data gathered through MS assays showed a correlation between a clustering of proteins from different snake species of the <em>Bothrops</em> genus and their phylogenetic classification (<a href="publications.html#Andrade-Silva:2016">Andrade-Silva et al., 2016</a>). However, the overrepresentation of some species in the protein sequence database used for the computational identification of peptides could incur in a bias in such type of analysis. To mitigate that problem, more recently it was generated phyloproteomic trees based on a <em>de novo</em> sequencing, that is, a peptide identification without the usage of a database. Nonetheless, that approach generated many false positives, which also are a source of noise in those analyses. Therefore, we are trying to circumvent those issues through the direct usage of MS raw data for the estimation of phyloproteomic trees. To this end, we are using a multiresolution-based approach for the construction of matrices as a function of fragment retention time, mass/charge ratio and intensity detection. Such matrices are being used to generate phyloproteomic trees through a Bayesian inference approach, using Markov Chain Monte Carlo methods. Finally, we are applying the CADM statistical test for the comparison of the cladograms yielded by our methodology with the one obtained using mitochondrial DNA. With this new methodology fully developed, we expect more robust results regarding to which extent the venom proteomic profiles matches the phylogeny of different species of <em>Bothrops</em> snakes.  
    </div>
  </div>
  <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#gvieira">Guilherme Vieira</a> - Undergrad Student <br/><br/>
    <a href="members.html#gmaciel">Gustavo Maciel</a> - Undergrad Student (Former Member)<br/><br/>
    <a href="members.html#vwraposo">Victor Raposo</a> - Undergrad Student (Former Member)<br/><br/>
    <a href="http://lattes.cnpq.br/9362702262041806" target="_blank">Solange M.T. Serrano, PhD</a> - Associate Investigator<br/>
    <em>(Principal Investigator at Applied Toxinology Laboratory, Butantan Institute)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4540289047342316" target="_blank">Carolina Brás</a> - Doctoral Student under Dr. Serrano supervision<br/><br/>
    <a href="http://lattes.cnpq.br/0554948941900699" target="_blank">Débora Andrade-Silva</a> - Doctoral Student under Dr. Serrano supervision<br/><br/>
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    An intermediate stage of this project was supported by <a href="https://bv.fapesp.br/en/bolsas/178365/computational-phylogenetic-analysis-of-bothrops-snakes-from-venom-proteomes/" target="_blank">the Undergrad Research scholarship # 2018/06682-0 from FAPESP</a>.
    </div>
  </div>
</div>

<br/><hr>


<div class="container" id="Scientiometer">
  <div class="row">
    <div class="twelve columns">
    <h6><b>Development of an information system for the gathering and analysis of scientific production reports</b></h6>

    Productivity assessment is an essential part of the monitoring and control of the scientific knowledge production. The Center of Scientific Development (<em>Centro de Desenvolvimento Científico</em> - CDC) of Butantan Institute prepares every year a detailed productivity report, based on information provided by researchers from all laboratories under CDC management. However, the current procedures for the report production, despite recent improvements, require laborious work from both researchers and members of the committee nominated by CDC to compile and analyze such data. Therefore, in this project we propose to settle that question through the design and implementation of a relational database, populating it with data from the current report and also from the ones of previous years. In a second (and current) stage, we are developing an information system for the report data gathering and automated upload of them into the aforementioned database. Finally, we plan to develop a user-friendly interface for the data analysis, including graph plotting of a number of scientometric indices. Importantly, we are making available the information system as free and open source, which will enable the usage of that system in other Research Institutes from São Paulo state and also in other institutions.
    </div>
  </div>
  <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#bscholl">Bruno Scholl</a> - Undergrad Student <br/><br/>
    <a href="members.html#paraujo">Paulo Araujo</a> - Undergrad Student <br/><br/>
    <a href="http://lattes.cnpq.br/6181892645004033" target="_blank">Sandra Vessoni, PhD</a> - Associate Investigator<br/>
    <em>(CDC Director)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4253838966347741" target="_blank">Maria Carolina Elias, PhD</a> - Associate Investigator<br/>
    <em>(CDC 1st Deputy Director)</em><br/><br/>
    <a href="http://lattes.cnpq.br/6244958363197940" target="_blank">Ivo LeBrun, PhD</a> - Associate Investigator<br/>
    <em>(CDC 2nd Deputy Director)</em><br/><br/>
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    This project is being supported by an Undergrad Research scholarship from Fundação Butantan.
    </div>
  </div>
</div>

</div>

<br/><br/>

<div class="container">
<div class="twelve columns" style="margin-top: 2%">
  <h5>Software</h5>

To carry out the aforementioned research projects, our group develop a myriad of software, using different programming languages and techniques. We list below some of them, including reference for the project they are related and a link for the source code repository (<em>importantly, our programs are always available as open and free software, since we believe that is a good practice in Science</em>). As in the case of the research projects, you can <a href="contact.html">contact us</a> if you are interested to contribute to the further development of any of those software and/or the tailoring of them to your problem.
  
<br/><br/>
  
</div>

<div class="container">
  <div class="row">
    <div class="twelve columns">
    <h5><strong>featsel</strong></h5>
    
    <img src="images/featsel_logo.png" class="software_logo" alt="featsel logo"><br/><br/>
    
    <em>featsel</em> is a framework especially designed for the benchmarking of feature selection algorithms and cost functions. Through a clean object-oriented structure and coded in C++, this framework allows the user to easily include new algorithms and/or cost functions and to carry out fast computational experiments. Moreover, <em>featsel</em> allows the user to take advantage of the fact that the search space of a feature selection procedure can be organized into a Boolean lattice; to this end, several methods are included in this framework especially for Boolean manipulation of feature subsets. <em>featsel</em> is also accompanied by several scripts to assist the user in the inclusion and removal of algorithms and cost functions, to carry out benchmarking experiments and to create tables and graphs with the obtained results.
    
    <br/><br/>
    
    The <em>featsel</em> framework architecture was firstly presented by <a href="publications.html#Reis:2017">Reis et al. (2017)</a>, and was used to generate results of the computational experiments presented in <a href="publications.html#Reis:2019">Reis et al. (2019)</a>. Currently, it is being used in the project "<a href="projects-and-software.html#modelSelection">Development of ABC-based approaches to tackle the lack of isolation problem in signaling pathway</a>".
    
    <br/><br/>

    <strong>Source code license:</strong> <a href="http://www.gnu.org/licenses" target="_blank">GNU GPL-3.0</a>.

    <br/><br/>

    <strong>Source code availability:</strong> <a href="https://github.com/msreis/featsel" target="_blank">github.com/msreis/featsel</a>.
    
    </div>
  </div>
</div>

<br/><hr>

<div class="container">
  <div class="row">
    <div class="twelve columns">
    <h5><strong>ReDyMo</strong></h5>
    
    <img src="images/ReDyMo_logo.jpeg" class="software_logo" alt="MITE logo"><br/><br/>

The Replication Dynamic Model (ReDyMo) is a stochastic dynamic model simulator that reproduces the DNA replication process of trypanosomatids. Initially, we focused on <em>Trypanosoma brucei</em> strain TREU927. This simulator is capable of:
<br/><br/>

<ul>

    <li> Calculation of various relevant properties about the replication process, including S-phase duration, mean inter-origin distance between replication origin sites, and the achieved replication percentage given a time limit;</li>
    
    <li> Storage of information about each step of the simulation, such as the replication stage at each model simulation iteration and whether any collisions happened during a given iteration;</li>
    
    <li>Storage of information regarding the replication timing program, that is, for each base pair it is recorded at which simulation iteration it was replicated;</li>
    
    <li>Simulation of replication-transcription conflicts with a variety of input parameters like replication machinery speed and transcription frequency;</li>

    <li>Simulation of dormant origins firing.</li>
</ul>
    
ReDyMo was originally coded in Python and later was ported to a high-performance C++ version. The architecture of that simulator is described in <a href="publications.html#daSilva:2019">da Silva et al. (2019)</a>. This simulator is still in use at the project "<a href="projects-and-software.html#replicationModels">DNA replication programming in trypanosomatids</a>".
    
            <br/><br/>

        <strong>Source code license:</strong> <a href="http://www.gnu.org/licenses" target="_blank">GNU GPL-3.0</a>.

                <br/><br/>

        <strong>Source code availability:</strong><br/>
        <a href="https://github.com/msreis/ReDyMo" target="_blank">github.com/msreis/ReDyMo</a> (Python version)
        <br/>
        <a href="https://github.com/msreis/ReDyMo-CPP" target="_blank">github.com/msreis/ReDyMo-CPP</a> (C++ version)
    </div>
  </div>
</div>

<br/><hr>

<div class="container">
  <div class="row">
    <div class="twelve columns">
    <h5><strong>MITE</strong></h5>

    <img src="images/MITE_logo.jpeg" class="software_logo" alt="MITE logo"><br/><br/>
            
    The Matrix of Intensity by Time of Elution (MITE) is a collection of programs for the designing of phyloproteomic cladograms (trees) based on mass spectrometry-based proteomics data (either identified peptides or raw data) and using the Markov Chain Monte Carlo (MCMC) procedure implemented in the <a href="https://nbisweden.github.io/MrBayes/index.html" target="_blank">MrBayes software</a>. Those programs were coded in Python and include procedures to prepare the NEXUS files necessary for the MCMC method and also the CADM statistical test for comparison of the yielded cladograms.
    
    <br/><br/>
    
    Details about the programs that compose MITE can be found at <a href="https://linux.ime.usp.br/~vwraposo/mac0499/" target="_blank">Victor Raposo's term paper</a> and also at <a href="https://linux.ime.usp.br/~mergipe/mac0499/" target="_blank">Gustavo Maciel's term paper</a>. This software is being developed in the project "<a href="projects-and-software.html#phyloproteomics">Phyloproteomic analysis of snake venom proteomes yielded by mass spectrometry-based assays</a>".
    
        <br/><br/>

        <strong>Source code license:</strong> <a href="http://www.gnu.org/licenses" target="_blank">GNU GPL-3.0</a>.

        <br/><br/>

        <strong>Source code availability:</strong> <a href="https://github.com/msreis/MITE" target="_blank">github.com/msreis/MITE</a>.

    </div>
  </div>
</div>

<br/><hr>

<div class="container">
  <div class="row">
    <div class="twelve columns">
    <h5><strong>Scientiometer</strong></h5>

        <img src="images/Scientiometer_logo.svg" class="software_logo" alt="Scientiometer logo"><br/><br/>

        Scientiometer is a platform for scientometric analyses of research institutions. Although Scientiometer was originally designed for evaluation of researchers and scientific production at Butantan Institute, this software may be employed for evaluation of any other institution. Scientiometer is being implemented using Ruby on Rails and also the Vue.js web framework.
      
        <br/><br/>
      
        The architecture of Scientiometer initial components (e.g., relational database and report submission system) is described in <a href="https://www.linux.ime.usp.br/~brunobbs/mac0499/" target="_blank">Bruno Scholl's term paper</a> (in Portuguese). This software is used in the project "<a href="projects-and-software.html#Scientiometer">Development of an information system for the gathering and analysis of scientific production reports</a>".
      
        <br/><br/>

        <strong>Source code license:</strong> <a href="http://www.gnu.org/licenses" target="_blank">GNU GPL-3.0</a>.

        <br/><br/>

        <strong>Source code availability:</strong> <a href="https://github.com/msreis/Scientiometer" target="_blank">github.com/msreis/Scientiometer</a>.

    </div>
  </div>
</div>

</div>

<br/><br/>

<footer>
  <div class="container">
        <i><a href="index.html">Dynamic Systems Biology Group</a>, <a href="http://butantan.gov.br/instituto-butantan/about-us" target="_blank">Butantan Institute</a><br/>
        Phone: +55-11-2627-3755, Fax: +55-11-2627-9731<br/>
        marcelo dot reis at butantan dot gov dot br</i>
    </div>
</footer>
  
<script>
  function openDrawerMenu(){
    var x = document.getElementById("mainNavBar");
    if (x.className === "navBar"){
      x.className += " responsive";
    } else {
      x.className = "navBar";
    }
  }
</script>


<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->


</body>

</html>


