<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Dynamic Systems Biology Group</title>
  <meta name="description" content="Dynamic Systems Biology Group">
  <meta name="author" content="Marcelo S. Reis">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="icon" type="image/svg" href="images/king-of-hearts.svg">

</head>


<body>


  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="twelve columns" style="margin-top: 2%">
        <h4>Dynamic Systems Biology Group</h4>
    </div>
  </div>


<div class="navBar" id="mainNavBar">
  <a href="index.html">About</a>
  <a href="members.html">Members</a>
  <a href="publications.html">Publications</a>
  <a href="projects.html">Projects</a>
  <a href="software.html">Software</a>
  <a href="teaching.html">Teaching</a>
  <a href="contact.html">Contact</a>
  <a href="javascript:void(0);" class="icon" onClick="openDrawerMenu()">&#9776;</a>
  <!--&#9776; is the code for the 3 line menu button-->
</div>

  <div class="container">
    <div class="row" style="margin-top: 2%">
    
        <h5>Ongoing projects</h5>
 
 In this page we list the main research projects that are currently in execution by our group. If you are interested in working on one of them, either as a student or a collaborator, do not hesitate <a href="contact.html">to contact us!</a>

<br/><br/>

<ul>
  <li><a href="projects.html#modelSelection">Development of Bayesian approaches to tackle the lack of isolation problem in signaling pathway</a><br/></li>
  
  <li><a href="projects.html#Goldilocks">Finding the "Goldilocks zones" of cell signaling pathways in cancer therapy</a><br/></li>
  
  <li><a href="projects.html#replicationModels">Stochastic models of DNA replication programming in trypanosomatids</a><br/></li>
  
  <li><a href="projects.html#phyloproteomics">Unsupervised learning of snake venom proteomes yielded by mass spectrometry-based assays</a><br/></li>
  
  <li><a href="projects.html#scientometrics">Development of an information system for the gathering and analysis of scientific production reports</a></li>
</ul>


<hr/>

    
<div class="container" id="modelSelection">
  <div class="row">
    <div class="twelve columns">
    <h6><strong>Development of Bayesian approaches to tackle the lack of isolation problem in signaling pathway</strong></h6>
    Dynamic models based on ordinary differential equations (ODEs) are useful tools for investigation of cell signaling pathways. Such models allow us to make predictions about the pathway behavior under different stimuli. ODE-based models are often calibrated using experimental data obtained by probing one or more species at several time points after a given cell stimulation. To this end, Bayesian approaches such as the approximate Bayesian computation (ABC) and the extended Kalman filter (EKF) are relevant tools, since they allow the assessment of the uncertainty in the estimation and also are a natural way to carry out model selection. In general, those models are estimated assuming that they are relatively isolated systems with known inputs; however, in many practical situations, we do not have any guarantee that that assumption holds. Therefore, in this project, we propose to tackle the lack of isolation problem in signaling pathway modeling through the development of a new approach based on ABC and/or EKF, which would take into account that model inputs also must be inferred. To this end, we also intend to use the reduction of the model selection problem to the feature selection problem that was recently developed by our group. Finally, we plan to test the new method on artificial instances and also on experimental measurements that are yielded in cancer cells studies at our lab. We expect that the proposed method will become a relevant tool in the studying of cell signaling pathways in many biological contexts, specially in cancer biology studies.
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#rnogueira">Ronaldo Nogueira</a> - PhD Student <br/><br/>
    <a href="members.html#mbatista">Marcelo Batista</a> - MSc Student <br/><br/>
    <a href="members.html#wwang">Willian Wang</a> - Undergrad Student <br/><br/>
    <a href="members.html#gestrela">Gustavo Estrela</a> - Former MSc Student, Currently Outside Collaborator<br/><br/>
    <a href="https://www.mpibpc.mpg.de/liepe" target="_blank">Juliane Liepe, PhD</a> - Associate Investigator<br/>
    <em>(Group Leader at Quantitative and Systems Biology, Max Planck Institute for Biophysical Chemistry, Germany)</em><br/><br/>
    <a href="http://lattes.cnpq.br/8341719454584768" target="_blank">Prof. Hugo A. Armelin</a> - Associate Investigator<br/>
    <em>(Coordinator of the <a href="http://www.cetics.com.br/en/" target="_blank">FAPESP RIDC CeTICS center</a>, Butantan Institute)</em><br/>
    </div>
  </div>

  <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    This project is currently supported by a PhD scholarship from CAPES. Initial parts of this work were supported by <a href="https://bv.fapesp.br/en/bolsas/175684/identification-of-cell-signaling-pathways-based-on-biochemical-reaction-kinetics-repositories/" target="_blank">the MSc scholarship # 2017/20575-9 from FAPESP</a>, and also by <a href="https://bv.fapesp.br/en/bolsas/189321/an-approximate-bayesian-computation-based-approach-to-tackle-the-lack-of-isolation-problem-in-signal/" target="_blank">the BPE grant # 2019/20025-4 from FAPESP</a>; activities of this latter grant were carried out by the Coordinator at Dr. Liepe's group in Göttingen, Germany.
    </div>
  </div>

  <div class="row">
    <div class="twelve columns">
    <br/><b>CNPq Research Group</b> <br/><br/>
    This project is registered at CPNq as the research line "<a href="http://dgp.cnpq.br/dgp/espelholinha/97618317029470061121430" target="_blank">Modelagem de vias de sinalização celular no contexto da biologia do câncer</a>", which in turn is under the research group "<a href="http://dgp.cnpq.br/dgp/espelhogrupo/660447" target="_blank">Controle do ciclo celular e da diferenciação celular</a>".
    </div>
  </div>

</div>

<br/><hr>

<br/>
 
 </div>
 
 <div class="container" id="Goldilocks">
  <div class="row">
    <div class="twelve columns" >
    <h6><strong>Finding the "Goldilocks zones" of cell signaling pathways in cancer therapy</strong></h6>
 
 The increase of cancer cell fitness during oncogenesis is accompanied by an increase of several cellular stresses, a phenomenon that may be exacerbated through mitogenic overstimulation. Recently, our group reported an unusual approach to cancer therapy, which relies on mitogenic overstimulation followed by inhibition of stress mitigation pathways. Although mitogenic overstimulation of cancer cells can often be achieved using natural mitogens, which generally are harmless for healthy cells, some types of cancer do not respond well to them. In this case, it might be necessary interventions downstream of mitogenic signaling pathways, a procedure that might also be cytotoxic to healthy cells. Therefore, we hypothesize the existence of a Goldilocks zone, that is, a just right set of such interventions that maximizes treatment efficacy at the same time sparing healthy cells from deleterious effects. To test this hypothesis, we propose to develop a methodology that encompasses: i) a Bayesian model selection procedure that draws candidate models from a reactions database and takes into account the lack of isolation problem that arises during the estimation of signaling pathways models; ii) the designing of classifiers of cell proliferation as a function of signaling pathways dynamics; iii) the incorporation of (i) and (ii) into a combinatorial optimization search for best sets of interventions. To test the proposed methodology, we plan to apply it into synthetic data and on results from experimental assays using cancer and non-tumorigenic cell lines. We also intend to experimentally validate promising predictions obtained with those wet lab assays. Testing this Goldilocks zones hypothesis is a timely endeavor, which could assist the discovery of new mitogenic signaling pathway activators for that novel cancer therapy approach.
  </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#rnogueira">Ronaldo Nogueira</a> - PhD Student <br/><br/>
    <a href="members.html#mbatista">Marcelo Batista</a> - MSc Student <br/><br/>
    <a href="members.html#csampaio">Cássia Sampaio</a> - Technical Training Student <br/><br/>
    <a href="members.html#fmontoni">Fabio Montoni</a> - Technical Training Student <br/><br/>
    <a href="members.html#wwang">Willian Wang</a> - Undergrad Student <br/><br/>
    <a href="https://www.mpibpc.mpg.de/liepe" target="_blank">Juliane Liepe, PhD</a> - Associate Investigator<br/>
    <em>(Group Leader at Quantitative and Systems Biology, Max Planck Institute for Biophysical Chemistry, Germany)</em><br/><br/>
    <a href="http://lattes.cnpq.br/8341719454584768" target="_blank">Prof. Hugo A. Armelin</a> - Associate Investigator<br/>
    <em>(Coordinator of the <a href="http://www.cetics.com.br/en/" target="_blank">FAPESP RIDC CeTICS center</a>, Butantan Institute)</em><br/><br/>
    <a href="http://www.ime.usp.br/~jb" target="_blank">Prof. Junior Barrera</a> - Associate Investigator<br/>
    <em>(Director of Institute of Mathematics and Statistics, University of São Paulo)</em><br/><br/>
    <a href="http://www.ime.usp.br/~ronaldo" target="_blank">Prof. Ronaldo F. Hashimoto</a> - Associate Investigator<br/>
    <em>(Associate Professor at Institute of Mathematics and Statistics, University of São Paulo)</em><br/><br/>
    <a href="http://lattes.cnpq.br/8578945562332621" target="_blank">Rosangela Wailemann, PhD</a> - Postdoc fellow under Prof. Armelin supervision <br/><br/>
    <a href="http://lattes.cnpq.br/0955478571926363" target="_blank">Edmilson Ozorio, PhD</a> - Postdoc fellow under Prof. Armelin supervision (Former Member)<br/>
    </div>
  </div>
  <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    This project is currently supported by <a href="https://bv.fapesp.br/en/auxilios/106474/finding-the-goldilocks-zones-of-cell-signaling-pathways-in-cancer-therapy/" target="_blank">the Regular Grant # 2019/21619-5 from FAPESP</a>, and also by a PhD scholarship from CAPES.
    </div>
  </div>
  
  <div class="row">
    <div class="twelve columns">
    <br/><b>CNPq Research Group</b> <br/><br/>
    This project is registered at CPNq as the research line "<a href="http://dgp.cnpq.br/dgp/espelholinha/97618317029470061121430" target="_blank">Modelagem de vias de sinalização celular no contexto da biologia do câncer</a>", which in turn is under the research group "<a href="http://dgp.cnpq.br/dgp/espelhogrupo/660447" target="_blank">Controle do ciclo celular e da diferenciação celular</a>".
    </div>
  </div>
  
  
  
</div>

<br/><hr>

<div class="container" id="replicationModels">
  <div class="row">
    <div class="twelve columns">
    <h6><strong>Stochastic models of DNA replication programming in trypanosomatids</strong></h6>
    
    <img src="images/T_cruzi.jpg" class="software_logo" alt="T. cruzi"><br/><br/>

    
    Trypanosomatids are protozoan that bear a peculiar genomic topology, with genes organized into polycistrons and constitutive transcription along the whole cell cycle. Given those properties, one hypothesis is that conflicts between DNA replication and transcription machineries contribute for the firing of replication origins along the cell cycle S phase in those organisms. In fact, based on a DNA replication model proposed for human cells, we developed a DNA replication stochastic dynamic model for <em>Trypanosoma brucei</em> that was calibrated with published MFA-seq data and used it to predict that increasing constitutive transcription levels indeed incur in an increase of fired origins; however, such increase has little impact on the overall S phase duration (<a href="publications.html#daSilva:2019">da Silva et al., 2019</a>). Those predictions were validated experimentally in alpha-amanitin-based transcription inhibition assays (<a href="publications.html#daSilva:2019">da Silva et al., 2019</a>). Nevertheless, the aforementioned hypothesis was not explored in other trypanosomatids, especially in <em>T. cruzi</em>. Recently, Dr. Maria Carolina Elias' group concluded the DNA origins mapping for <em>T. cruzi</em> through MFA-seq assays. Those assays unveiled that several origins are located in coding regions of the dispersed gene family 1 (DFG-1) genes, a gene family relevant for the parasite life cycle. Once DFG-1 genes have high genetic variability, one possibility is that conflicts between DNA replication and transcription machineries are responsible for the verified genetic variability. If this latter assumption is correct, one corollary is that the origin firing distribution in <em>T. cruzi</em> is conditioned by the parasite's own genomic organization. Therefore, through a combination of dry and wet lab assays, we seek in this project to answer the following questions: 
    
    <br/><br/>
    
    <ul> 
      <li> <em> Do conflicts between DNA replication and transcription machineries induce increased origin firing in T. cruzi?</em> </li>
      <li> <em> Is the origin firing distribution in <em>T. cruzi</em> conditioned by the genomic organization?</em></li>
    </ul>

    Apart from the resolution of the aforementioned questions, we aim with this project to improve computational dynamic models of the DNA replication programming. We also expect theoretical advances (e.g., closed solutions of stochastic processes that describe DNA replication dynamics in trypanosomatids), which would be derived through analyses of computational assays.
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#bscholl">Bruno Scholl</a> - MSc Student <br/><br/>
    <a href="members.html#lcorunha">Ligia Corunha</a> - Undergrad Student <br/><br/>
    <a href="members.html#vhariki">Victor Hariki</a> - Former Undergrad Student, Currently Outside Collaborator <br/><br/>
    <a href="http://lattes.cnpq.br/4253838966347741" target="_blank">Maria Carolina Elias, PhD</a> - Associate Investigator<br/>
    <em>(Director of Cell Cycle Laboratory, Butantan Institute)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4775859232176039" target="_blank">Marcelo Santos, PhD</a> - Postdoc fellow under Dr. Elias supervision (Former Member)<br/><br/>
    <a href="members.html#gcayres">Gustavo Cayres</a> - Undergrad Student (Former Member)<br/>
    </div>
  </div>
  
  <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    Initial stages of this project were supported by <a href="https://bv.fapesp.br/en/bolsas/168770/design-and-simulation-of-computational-models-of-the-dna-replication-dynamics-in-kinetoplastids/" target="_blank">the Undergrad Research scholarship # 2016/17775-3 from FAPESP</a>. Currently, its execution has a financial support from CNPq.
    </div>
  </div>
  
  <div class="row">
    <div class="twelve columns">
    <br/><b>CNPq Research Group</b> <br/><br/>
    This project is registered at CPNq as the research line "<a href="http://dgp.cnpq.br/dgp/espelholinha/97618317029470061121369" target="_blank">Decifrando a dinâmica de replicação em Trypanosomas</a>", which in turn is under the research group "<a href="http://dgp.cnpq.br/dgp/espelhogrupo/660447" target="_blank">Controle do ciclo celular e da diferenciação celular</a>".
    </div>
  </div>
  
</div>

<br/><hr>


<div class="container" id="phyloproteomics">
  <div class="row">
    <div class="twelve columns">
    <h6><b>Unsupervised learning of snake venom proteomes yielded by mass spectrometry-based assays</b></h6>

    Snake venoms are complex mixtures composed mostly of proteins and peptides. One analytical strategy that is often used to study them is the mass-spectrometry (MS)-based proteomics. Recent analysis using proteomic data gathered through MS assays showed a correlation between a clustering of proteins from different snake species of the <em>Bothrops</em> genus and their phylogenetic classification (<a href="publications.html#Andrade-Silva:2016">Andrade-Silva et al., 2016</a>). However, the overrepresentation of some species in the protein sequence database used for the computational identification of peptides could incur in a bias in such type of analysis. To mitigate that problem, more recently it was generated phyloproteomic trees based on a <em>de novo</em> sequencing, that is, a peptide identification without the usage of a database. Nonetheless, that approach generated many false positives, which also are a source of noise in those analyses. Therefore, we are trying to circumvent those issues through the direct usage of MS raw data for the estimation of phyloproteomic trees. To this end, we are using a multiresolution-based approach for the construction of matrices as a function of fragment retention time, mass/charge ratio and intensity detection. Such matrices are being used to generate phyloproteomic trees through Unsupervised Learning approaches, for instance, through agglomerative clusterings. Finally, we are applying the CADM statistical test for the comparison of the cladograms yielded by our methodology with the one obtained using mitochondrial DNA. With this new methodology fully developed, we expect more robust results regarding to which extent the venom proteomic profiles matches the phylogeny of different species of <em>Bothrops</em> snakes, an assessment that is critical in evolutionary studies of those organisms.  
    </div>
  </div>
  <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#gvieira">Guilherme Vieira</a> - Former Undergrad Student, Currently Outside Collaborator <br/><br/>
    <a href="members.html#gmaciel">Gustavo Maciel</a> - Undergrad Student (Former Member)<br/><br/>
    <a href="members.html#vwraposo">Victor Raposo</a> - Undergrad Student (Former Member)<br/><br/>
    <a href="http://lattes.cnpq.br/9362702262041806" target="_blank">Solange M.T. Serrano, PhD</a> - Associate Investigator<br/>
    <em>(Principal Investigator at Applied Toxinology Laboratory, Butantan Institute)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4540289047342316" target="_blank">Carolina Brás</a> - Doctoral Student under Dr. Serrano supervision<br/><br/>
    <a href="http://lattes.cnpq.br/0554948941900699" target="_blank">Débora Andrade-Silva, MSc</a> - Doctoral Student under Dr. Serrano supervision<br/>
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    An intermediate stage of this project was supported by <a href="https://bv.fapesp.br/en/bolsas/178365/computational-phylogenetic-analysis-of-bothrops-snakes-from-venom-proteomes/" target="_blank">the Undergrad Research scholarship # 2018/06682-0 from FAPESP</a>.
    </div>
  </div>
</div>

<br/><hr>


<div class="container" id="scientometrics">
  <div class="row">
    <div class="twelve columns">
    <h6><b>Development of an information system for the gathering and analysis of scientific production reports</b></h6>

    Productivity assessment is an essential part of the monitoring and control of the scientific knowledge production. The Center of Scientific Development (<em>Centro de Desenvolvimento Científico</em> - CDC) of Butantan Institute prepares every year a detailed productivity report, based on information provided by researchers from all laboratories under CDC management. However, the current procedures for the report production, despite recent improvements, require laborious work from both researchers and members of the committee nominated by CDC to compile and analyze such data. Therefore, in this project we propose to settle that question through the design and implementation of a relational database, populating it with data from the current report and also from the ones of previous years. In a second (and current) stage, we are developing an information system for the report data gathering and automated upload of them into the aforementioned database. Finally, we plan to develop a user-friendly interface for the data analysis, including graph plotting of a number of scientometric indices. Importantly, we are making available the information system as free and open source, which will enable the usage of that system in other Research Institutes from São Paulo state and also in other institutions.
    </div>
  </div>
  <div class="row">
    <div class="twelve columns">
    <br/><b>Team</b> <br/><br/>
    <a href="members.html#msreis">Marcelo S. Reis, PhD</a> - Coordinator <br/><br/>
    <a href="members.html#bscholl">Bruno Scholl</a> - MSc Student <br/><br/>
    <a href="members.html#paraujo">Paulo Araujo</a> - Former Undergrad Student, Currently Outside Collaborator <br/><br/>
    <a href="http://lattes.cnpq.br/6181892645004033" target="_blank">Sandra Vessoni, PhD</a> - Associate Investigator<br/>
    <em>(CDC Director)</em><br/><br/>
    <a href="http://lattes.cnpq.br/4253838966347741" target="_blank">Maria Carolina Elias, PhD</a> - Associate Investigator<br/>
    <em>(CDC 1st Deputy Director)</em><br/><br/>
    <a href="http://lattes.cnpq.br/6244958363197940" target="_blank">Ivo LeBrun, PhD</a> - Associate Investigator<br/>
    <em>(CDC 2nd Deputy Director)</em><br/>
    </div>
  </div>
   <div class="row">
    <div class="twelve columns">
    <br/><b>Financial Support</b> <br/><br/>
    This project is being supported by an Undergrad Research scholarship from Fundação Butantan.
    </div>
  </div>
</div>

</div>

<br/><br/>

<footer>
  <div class="container">
        <i><a href="index.html">Dynamic Systems Biology Group</a>, <a href="http://butantan.gov.br/instituto-butantan/about-us" target="_blank">Butantan Institute</a><br/>
        Phone: +55-11-2627-3755, Fax: +55-11-2627-9731<br/>
        marcelo dot reis at butantan dot gov dot br</i>
    </div>
</footer>
  
<script>
  function openDrawerMenu(){
    var x = document.getElementById("mainNavBar");
    if (x.className === "navBar"){
      x.className += " responsive";
    } else {
      x.className = "navBar";
    }
  }
</script>


<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->


</body>

</html>


